lxd/vm_qemu.go

```diff
@@ -512,12 +512,7 @@ func (vm *vmQemu) Start(stateful bool) error {
 	// Copy OVMF settings firmware to nvram file.
 	// This firmware file can be modified by the VM so it must be copied from the defaults.
 	if !shared.PathExists(vm.getNvramPath()) {
```

Comment:
@stgraber is this line needed because the nvram file can be modified from inside the VM guest, and so it will persist changes between reboots? I was prompted to ask this question because I saw the part you added to the Update() function to regenerate the nvram file if the security.secureboot setting change.

Reply:
Correct, we absolutely don't want to re-gen on every boot as that would potentially cause data/settings loss.

When the user flips the `secureboot` setting, we have no choice but to re-generate and hope that the user understands what they're doing.